RussianFilter/sections/adservers.txt

```diff
@@ -4412,7 +4412,7 @@ us6odtasve.ru
 ||vsekiski.org^$third-party
 ||vsem.ru/img$third-party
 ||vsemayki.ru*/?ref=$third-party
-||vsemayki.ru/*ref=$third-party
```

Comment:
@Alex-302 do we need these rules?

Reply:
@zloyden Not sure. 
can I delete these rules?

Reply:
Well, you can temporary add `||vsemayki.ru/*ref=$third-party,badfilter` to the User rules.
But these rules are really old and @Alex-302 knows more about them. Let's wait for his decision.

Reply:
The rulses are old, but they was added to block ads from `vsemayki.ru`

Reply:
Ok.

Reply:
@akeinhell update should be available in 1-2 hours.

Reply:
Thanks